R/key-helpers.R

```diff
@@ -62,16 +62,19 @@ cdm_check_constraints <- function(dm) {
 check_key <- function(.data, ...) {
   data_q <- enquo(.data)
   .data <- eval_tidy(data_q)
-  args <- exprs(...)
-  names(args) <- set_names(paste0("...", seq_along(args)))
+
+  cols_avail <- colnames(.data)
+  # if no column is chosen, all columns are used for the check
+  cols_chosen <- if (is_empty(enexprs(...))) cols_avail else tidyselect::vars_select(cols_avail, ...)
```

Comment:
Do we even need the special behavior for an empty argument list here? Can the user ask `check_key(everything())` ?

Reply:
1. not necessarily (it would be a quick way to check if the combination of all columns is a unique key though)
2. yes

Reply:
Let's keep default semantics then.

Reply:
Just to make sure: this does mean I can merge without further changes, right?

Reply:
"Default semantics" means that we don't need to special-case behavior of `...` (but we need to watch out what happens with an empty selection). I'll tweak.